server/api/configuration/routes/get_configuration_status.js

```diff
@@ -17,12 +17,14 @@ module.exports = (server) => ({
     req.pre.auth0.rules.getAll()
       .then(rules => {
         const rule = _.find(rules, { name: 'auth0-authorization-extension' });
-        reply({
-          rule: {
-            exists: !!rule,
-            enabled: rule ? rule.enabled : false
-          }
-        });
+        return {
+          exists: !!rule,
+          enabled: rule ? rule.enabled : false
+        };
+      })
+      .then(rule => {
+        req.storage.getStatus()
+          .then(database => reply({ rule, database }));
```

Comment:
can you add a catch here and return 0 as the size if reading fails? we don't want to break the config endpoint if storage is somehow failing